react-stopwatch/my-app/src/StopWatch.tsx

```diff
@@ -0,0 +1,38 @@
+import { FaPlay } from 'react-icons/fa';
+import { FaPause } from 'react-icons/fa6';
+import { useState } from 'react';
+
+export function StopWatch() {
+  const [playing, setPlaying] = useState(false);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout>();
+  const [timeElapsed, setTimeElapsed] = useState(0);
+
+  function setIcon(): void {
```

Comment:
This is a strange name. A better name would be something like handlePlayPause, or handleClick.